src/binaryen-c.cpp

```diff
@@ -3245,6 +3247,26 @@ BinaryenModuleWrite(BinaryenModuleRef module, char* output, size_t outputSize) {
     .outputBytes;
 }
 
+size_t BinaryenModuleWriteSExpr(BinaryenModuleRef module,
+                                char* output,
+                                size_t outputSize) {
+
+  if (tracing) {
+    std::cout << "  // BinaryenModuleWriteSExpr\n";
+  }
+
+  // use a stringstream as an std::ostream. Extract the std::string
+  // representation, and then store in the output.
+  std::stringstream ss;
+  WasmPrinter::printModule((Module*)module, ss);
+
+  const std::string temp = ss.str();
```

Comment:
can use `const auto` here and below.

Reply:
Why `auto`? it's a simple type, and someone new to C++ many not know the type of `.str()`. I don't see a benefit of using `auto`.  I'm trying to understand the rationale here --- if this is the project style, then by all means, I'll change the code, but I see no advantage.

Reply:
Bump, @kripken  :)

Reply:
It is clear enough by convention that `.str()` returns a string, so it is worth saving on the visual complexity of this line.

Reply:
OK. changed to `const auto`.